.github/workflows/cross-version-tests.yml

```diff
@@ -185,6 +185,10 @@ jobs:
           pip install -U "pip$PIP_SPEC" wheel
           pip install -e .[extras]
           pip install -r requirements/test-requirements.txt
+      - name: Install tensorflow for keras
```

Comment:
looks like keras dev has some dependency on tensorflow:

https://github.com/mlflow/mlflow/actions/runs/7778504032/job/21208301502#step:15:58

Reply:
Issue:

If you install tensorflow, it will overwrite existing keras version to 2.15
see https://keras.io/getting_started/

Reply:
i think this should install it before, since it's before the `- name: Install ${{ matrix.package }} ${{ matrix.version }}` line

Reply:
Oh you are right ^^^, thanks :)

Reply:
The dev issue should be resolved by pointing keras-nightly to the github head. 

```
  package_info:
    pip_release: "keras"
    install_dev: |
      pip install --upgrade git+https://github.com/keras-team/keras.git
```

For some reason keras-nightly is pointing to keras 2.15